.github/workflows/robot/internal/bot/assign.go

```diff
@@ -50,10 +53,101 @@ func (b *Bot) Assign(ctx context.Context) error {
 }
 
 func (b *Bot) getReviewers(ctx context.Context) ([]string, error) {
+	// If a backport PR was found, assign original reviewers. Otherwise fall
+	// through to normal assignment logic.
+	if isBackport(b.c.Environment.UnsafeBase) {
+		reviewers, err := b.backportReviewers(ctx)
+		if err == nil {
+			return reviewers, nil
+		}
+		log.Printf("Assign: Found backport PR, but failed to find original reviewers: %v. Falling through to normal assignment logic.", err)
+	}
+
 	docs, code, err := b.parseChanges(ctx)
 	if err != nil {
 		return nil, trace.Wrap(err)
 	}
-
 	return b.c.Review.Get(b.c.Environment.Author, docs, code), nil
 }
+
+func (b *Bot) backportReviewers(ctx context.Context) ([]string, error) {
+	// Search inside the PR to find a reference to the original PR.
+	original, err := b.findOriginal(ctx,
+		b.c.Environment.Organization,
+		b.c.Environment.Repository,
+		b.c.Environment.Number)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+
+	var originalReviewers []string
+
+	// Append list of reviewers that have yet to submit a review.
+	reviewers, err := b.c.GitHub.ListReviewers(ctx,
+		b.c.Environment.Organization,
+		b.c.Environment.Repository,
+		original)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	for _, reviewer := range reviewers {
+		originalReviewers = append(originalReviewers, reviewer)
+	}
+
+	// Append list of reviews that have submitted a review.
+	reviews, err := b.c.GitHub.ListReviews(ctx,
+		b.c.Environment.Organization,
+		b.c.Environment.Repository,
+		original)
+	if err != nil {
+		return nil, trace.Wrap(err)
+	}
+	for _, review := range reviews {
+		originalReviewers = append(originalReviewers, review.Author)
+	}
+
+	return originalReviewers, nil
+}
+
+func (b *Bot) findOriginal(ctx context.Context, organization string, repository string, number int) (int, error) {
+	pull, err := b.c.GitHub.GetPullRequest(ctx,
+		organization,
+		repository,
+		number)
+	if err != nil {
+		return 0, trace.Wrap(err)
+	}
+
+	var original string
+
+	// Search inside the title and body for the original PR number.
+	title := pattern.FindStringSubmatch(pull.UnsafeTitle)
+	body := pattern.FindStringSubmatch(pull.UnsafeBody)
+	switch {
+	case len(title) == 0 && len(body) == 0:
+		return 0, trace.NotFound("no PR referenced in title or body")
+	case len(title) == 0 && len(body) == 2:
+		original = body[1]
+	case len(title) == 2 && len(body) == 0:
+		original = title[1]
+	case len(title) == 2 && len(body) == 2:
+		if title[1] != body[1] {
+			return 0, trace.NotFound("different PRs referenced in title and body")
```

Comment:
You could hit the API to see if they are both PRs. One might be an issue.

For example, something like this body would not work with the code as-is.

```
Backports #123 to branch/v8
Fixes #456
```